packages/liquidator/src/liquidator.js

```diff
@@ -265,9 +280,40 @@ class Liquidator {
         inputPrice: scaledPrice.toString()
       });
 
-      // we couldnt liquidate, this typically would only happen if our balance is 0
+      // we couldnt liquidate, this typically would only happen if our balance is 0 or the withdrawal liveness
+      // has not passed the WDF's activation threshold.
       // This gets logged as an event, see constructor
       if (!liquidationArgs) {
+        // If WDF is active but liveness hasn't passed activation %, then send customized log:
+        if (
+          this.defenseActivationPercent &&
+          !this.liquidationStrategy.utils.passedDefenseActivationPercent({ position, currentBlockTime })
+        ) {
+          // If `startingBlock` and `endingBlock` are specified and the requested withdrawal was within the
+          // block window, then upgrade the log level:
+          let logLevel = "debug";
+          if (!isNaN(this.startingBlock) && !isNaN(this.endingBlock)) {
+            const blockWindowHasRequestedWithdraw = await this.empContract.getPastEvents("RequestWithdrawal", {
+              fromBlock: this.startingBlock,
+              toBlock: this.endingBlock
+            });
+            if (blockWindowHasRequestedWithdraw) {
```

Comment:
tiny nit: This could be done more cleanly using a simply ternary operator.
```
const logLevel = blockWindowHasRequestedWithdraw ? "info" : "debug";
```

Feel free to ignore as well, this is more personal preference.